server/app/models/LifecycleStage.java

```diff
@@ -17,6 +18,13 @@ public enum LifecycleStage {
 
   private final String stage;
 
+  private static ImmutableMap<LifecycleStage, String> descriptions =
```

Comment:
issue: These values aren't really tied to the LifecycleStage model, and are only relevant when we're exporting applications, so I don't know that this is the right place for them.

issue: By declaring these as strings, we end up matching strings in a few different places, such as tests or when building the `ApplicationJsonExportData`.

suggestion: Can we declare this as an enum in services/export/enums, similar to the `SubmitterType` enum? And then call `.toString()` on it when we actually want to export it?

Reply:
I like the enum idea, and I belatedly remember you mentioning this at the beginning. Done.